artifactory_cleanup/rules/docker.py

```diff
@@ -12,6 +12,8 @@
 
 ctx_mgr_block, ctx_mgr_test = get_context_managers()
 
+MANIFEST_FILENAME = "manifest.json"
```

Comment:
Shall we move it to RuleForDocker class? So you can overwrite it by creating a subclass
```python
class RuleForDocker(Rule):
    MANIFEST_FILENAME = "manifest.json"

class DifferentRuleForDocker(RuleForDocker):
    MANIFEST_FILENAME = "list.manifest.json"
```

Reply:
Oops, I lost this change off my radar tbh :)

Okay, that makes sense - moved this into a class variable.